src/Job.php

```diff
@@ -85,6 +90,26 @@ public function delete(): void
         $this->consumer->acknowledge($this->message);
     }
 
+    /**
```

Comment:
Annotation

Reply:
add test for exception when passed something like `function (): void {}` or `\tmpfile()`

Reply:
Tests added